src/index.js

```diff
@@ -108,7 +109,7 @@ class Bitswap {
       (has, cb) => {
         this._updateReceiveCounters(peerId.toB58String(), block, has)
         if (has) {
-          return cb()
+          return nextTick(cb)
```

Comment:
In this case you're already on a future tick from the `blockstore.has` call above.

Reply:
I'd recommend we leave these in place to avoid the internals here being sync _or_ async depending on the existence of the block. We should try to keep the function footprint the same imo.